spring-kafka/src/test/java/org/springframework/kafka/listener/ConcurrentMessageListenerContainerTests.java

```diff
@@ -173,6 +174,48 @@ public void onPartitionsAssigned(Collection<TopicPartition> partitions) {
 	}
 
 	@Test
+	public void testAutoCommitWithCompositeStrategy() throws Exception {
```

Comment:
We don't have such a `strategy` any more. Therefore the test case reflect some parallel irrelevant reality.
Will fix on merge.